src/app/authentication/pages/login/login.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LoginComponent } from './login.component';
+
+describe('LoginComponent', () => {
+  let component: LoginComponent;
+  let fixture: ComponentFixture<LoginComponent>;
+  let baseElement: any;
+
+  const SELECTORS = {
+    headerLogo: '.header__logo',
+    headerTitle: '.header__subtitle'
+  };
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ LoginComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(LoginComponent);
+    component = fixture.componentInstance;
+    baseElement = fixture.nativeElement;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
@carryall What if we use the test context with out `should` ?
Ex: `creates a component`